### DISCUSSION TITLE
few information in documentation

### DISCUSSION THREAD

**Discussion by happyDuke96** [chatwoot/chatwoot#5934] [Chatwoot APIs] [answered]
hello everyone, you can give api to create a label, do not bind to a conversation, but simply create a label then bind to a conversation, the next question is how can I add a photo for client contacts

**Comment by happyDuke96** [comment]
Excuse me, of course, but I don’t often get an answer on this, I want to ask an additional question, can I send CSAT to the client via Api Channel

**Reply by sojan-official** [reply]
you can create an outgoing message with `content_type: input_csat` to create  CSAT messages

**Reply by happyDuke96** [reply]
works great, but now how to change the message in the widget and in the agent side?
![agen_side_image](https://user-images.githubusercontent.com/105979703/204458881-aeaef944-cae1-4874-b3a1-7bec8317af40.PNG)
![widget_image](https://user-images.githubusercontent.com/105979703/204458886-441e4809-2908-40fe-b314-8967532023e4.PNG)
I know that here I used a web inbox, but this is to visually see how it works, I don’t know much about it, I first have to test it myself and then prove to our company that this project can be implemented please help me and give me more information

**Comment by sojan-official** [comment, marked as answer]
@AbbosAbduhakimov  please refer : https://www.chatwoot.com/developers/api/ 
you can add labels to the conversation by calling https://www.chatwoot.com/developers/api/#tag/Conversation-Labels

**Reply by happyDuke96** [reply]
yes it works, I get ok status but this label is not displayed on the screen
![agent_side_image](https://user-images.githubusercontent.com/105979703/204460565-bd936c05-64ee-45a4-87c1-a3167db715d5.PNG)
![label_postman_image](https://user-images.githubusercontent.com/105979703/204460569-44ee6dda-fc4e-4d80-b4bb-0f003a1b9109.PNG)
Can I add this label via API?
![labels](https://user-images.githubusercontent.com/105979703/204460679-8a76c04c-a507-463a-a71a-81d1a0953031.PNG)
